torchaudio/functional.py

```diff
@@ -1317,7 +1317,6 @@ def phaser(
     delay_buf = torch.zeros(waveform.shape[0], delay_buf_len, dtype=dtype, device=device)
 
     mod_buf_len = int(sample_rate / mod_speed + .5)
-    mod_buf = torch.zeros(mod_buf_len, dtype=dtype, device=device)
```

Comment:
Is this not used?

Reply:
@mthrok , ```mod_buf``` is the output of ```_generate_wave_table``` . So removed this redundant intialization.